src/orderbook.js

```diff
@@ -1185,6 +1309,14 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   assert(!this._currencyGets.is_native() && !this._currencyPays.is_native(),
     'Autobridging is only for IOU:IOU orderbooks');
 
+  if (this._destroyed) {
+    return;
+  }
+
+  if (!this._gotOffersFromLegOne || !this._gotOffersFromLegTwo) {
```

Comment:
I looks like this function is only ever called when one of the leg books is updated, so always one or the other of these would be true